lib/commands/general.js

```diff
@@ -130,6 +130,52 @@ commands.mobileScrollToPage = async function (opts = {}) {
   });
 };
 
+
+/**
+ *  API to invoke methods defined in Android app.
+ *
+ *  Example data
+ *  {
+ *   target: 'activity',
+ *   methods:
+ *         [
+ *           {
+ *               name: "someMethod",
+ *           },
+ *           {
+ *               name: "anotherMethod",
+ *               args:
+ *                   [
+ *                       {value: "Lol", type: 'java.lang.CharSequence'},
+ *                       {value: 1, type: 'int'}
+ *                   ]
+ *           }
+ *         ]
+ * }
+ *
+ * In above example, method "someMethod" will be invoked on 'activity'. On the result, "anotherMethod" will be invoked
+ * - target can be either 'activity' or 'application'. If 'activity', it will try to invoke given method on current activity
+ *   If target is set to 'application', methods will be invoked on application class
+ *
+ * - Only Public methods can be invoked
+ * - following primitive types are supported: "int", "boolean", "byte", "short", "long", "float", "char"
+ * -  Non-primitive types with fully qualified name "java.lang.*" is also supported:
+ *                              Eg. "java.lang.CharSequence", "java.lang.String", "java.lang.Integer", "java.lang.Float",
+ *                              "java.lang.Double", "java.lang.Boolean", "java.lang.Long", "java.lang.Short",
+ *                              "java.lang.Character" etc...
+ *
+ *
+ * @throws  {Error} if target is not 'activity' or 'application'
+ * @throws  {Error} if a method is not found with given argument types
+ *
+ * @return will return execution results of method as string. If method return type is void, "<VOID>" will be returned
```

Comment:
`@return {*} the result of the last method in the invocation chain. If method return type is void, then "<VOID>" will be returned`